docs/UserGuide.md

```diff
@@ -47,19 +47,19 @@ App-Ointment is a desktop app for for managing and scheduling patient appointmen
 * Words in `UPPER_CASE` are the parameters to be supplied by the user.<br>
   e.g. in `add n/NAME`, `NAME` is a parameter which can be used as `add n/John Doe`.
 
-* Items in square brackets are optional.<br>
+* Parameters in square brackets are optional.<br>
   e.g `n/NAME [t/TAG]` can be used as `n/John Doe t/friend` or as `n/John Doe`.
 
-* Items with `…`​ after them can be used multiple times including zero times.<br>
+* Parameters with `…`​ after them can be used multiple times (including zero times).<br>
   e.g. `[t/TAG]…​` can be used as ` ` (i.e. 0 times), `t/friend`, `t/friend t/family` etc.
 
 * Parameters can be in any order.<br>
-  e.g. if the command specifies `n/NAME p/PHONE_NUMBER`, `p/PHONE_NUMBER n/NAME` is also acceptable.
+  e.g. if the command format specifies `n/NAME p/PHONE_NUMBER`, `p/PHONE_NUMBER n/NAME` is also acceptable.
 
 * If a parameter is expected only once in the command but you specified it multiple times, only the last occurrence of the parameter will be taken.<br>
   e.g. if you specify `p/12341234 p/56785678`, only `p/56785678` will be taken.
 
-* Extraneous parameters for commands that do not take in parameters (such as `help`, `list`, `exit` and `clear`) will be ignored.<br>
+* Extraneous parameters for commands that do not take in parameters (such as `clear`, `exit`, `help`, `list-appt`, `list-doctor` and `list-patient`) will be ignored.<br>
```

Comment:
this should be all 3 clears right? should we write somewhere in the UG that when we refer to like the front word w/o the back words, we refer to all 3 commands? ie. if we refer to `clear` without the `-xxxx`, we are referring to all 3 commands? or should we just be specific (and mildly repetitive) and just mention all 3?

Reply:
I think we should be specific, if not one might think `clear` or `list` on their own are valid commands. WIll add in the 3 clear commands.